Updating README.md

speco29 opened the pull request:
Added Labeler, Greeting, CodeQL, and Dependecies in the README.

speco29 committed:
Update README.md

Added Labeler, Greetings, CodeQL and Dependencies

speco29 committed:
Update README.md

Added Labeler, Greeting, CodeQL, and Dependecies in the README.

4darsh-Dev commented on README.md:
```diff
@@ -80,3 +80,10 @@ Website outline plan including webpages and their content:
 ### 10. Error Page (404)
 - Message indicating the page was not found
 - Link to redirect back to the Home page
+- # Community Website
```
Add the badges on top not bottom, 
rest is fine.

4darsh-Dev reviewed (changes requested):
everything is fine, 
just a minor change required 
fix it

speco29 committed:
Update README.md

Added Issue Labeler, PR Labeler Greetings, CodeQL and Dependencies in the README.md
